pandas/io/excel/_odfreader.py

```diff
@@ -0,0 +1,165 @@
+import pandas
+from pandas.io.parsers import TextParser
+
+
+class _ODFReader(object):
+    """Read tables out of OpenDocument formatted files
+
+    Parameters
+    ----------
+    filepath_or_stream: string, path to be parsed or
+        an open readable stream.
+    """
+    def __init__(self, filepath_or_stream):
+        try:
+            from odf.opendocument import load as document_load
+            from odf.table import Table
+        except ImportError:
+            raise ImportError("Install odfpy for OpenDocument support")
```

Comment:
Is there a minimum version required to match the structure of the imports being performed in this code?

Reply:
I'm not sure the exact version but as far as I can tell I started with a 1.3.x version and the API looks stable since then. Also pypi suggests that 1.2.0 was a "pre-release" https://pypi.org/project/odfpy/#history

I did add the text to match what's in _xlrd. so I'm going to mark this issue was resolved.